rollup.config.js

```diff
@@ -11,50 +11,25 @@ const external = [
 
 const watch = { exclude: ['node_modules/**'] };
 
-const babelConfig = {
-  // without this setting, asyncGenerator is included but not used (???)
-  externalHelpersWhitelist: [
-    'classCallCheck',
-    'inherits',
-    'possibleConstructorReturn',
-    'extends',
-  ],
-  exclude: 'node_modules/**',
-  babelrc: false,
-  presets: [
-    require.resolve('babel-preset-react'),
-    [
-      require.resolve('babel-preset-env'),
-      {
-        targets: { browsers: ['last 2 versions'] },
-        loose: true,
-        modules: false,
-        // debug: true,
-        exclude: [
-          // don't need fancy typeof
-          'transform-es2015-typeof-symbol',
-        ],
-      },
-    ],
-  ],
-  plugins: [
-    require.resolve('babel-plugin-transform-class-properties'),
-    require.resolve('babel-plugin-transform-object-assign'),
-    require.resolve('babel-plugin-external-helpers'),
```

Comment:
Babel is awesome, but it's nice to not have to worry about which not-yet-standardized features you're opting into.

Reply:
Yeah, in this case, Babel wasn’t doing anything, since TypeScript handles JSX, `Object.assign` (at least with JSX spreads), and ES6 classes.